hw2/05-form.js

```diff
@@ -1 +1,66 @@
 // Add your code here
+let submission = document.querySelector('form');
+
+submission.addEventListener('submit', (event) => {
+  const title = '========= Form Submission =========';
+  const userName = document.getElementById('fullName');
+  const name = userName.value;
```

Comment:
You can simplify this by saying:
`const name = document.getElementById('fullName').value || 'no submission';`
That way if the value is not provided then the default value is 'no submission' and you don't have to do as many if/else statements later. The rest of the form inputs can be simplified in a similar way.